pkg/loadbalancer/stream/loadbalancer.go

```diff
@@ -398,3 +426,21 @@ func (lb *LoadBalancer) removeFromPendingTarget(target types.Target) {
 func (lb *LoadBalancer) isStreamRunning() bool {
 	return lb.ctx != nil && lb.ctx.Err() == nil
 }
+
+// InterfaceCreated -
+// When a new NSM interface appears poke pending Targets to attempt
+// processing them
+func (lb *LoadBalancer) InterfaceCreated(intf networking.Iface) {
+	lb.mu.Lock()
+	if !lb.isStreamRunning() {
+		return
+	}
+	lb.mu.Unlock()
+	// send trigger if interface is of interest (i.e. lb NSE)
+	if strings.HasPrefix(intf.GetName(), GetInterfaceNamePrefix()) {
+		lb.pendingCh <- struct{}{}
+	}
+}
+
+func (lb *LoadBalancer) InterfaceDeleted(intf networking.Iface) {
```

Comment:
This would also need to be implemented, when an interface is removed, the routes of some targets might be removed. The verifyTargets has for responsibility to check that, clean them and add them back to the pending target list.